### PR TITLE
Add unit test for func IsGTE

### PR DESCRIPTION
**Pull request by Hellcatlk**
Signed-off-by: ZouYu <zouy.fnst@cn.fujitsu.com>

### CODE REVIEW

**Comment by johngmyers** [comment]
Thank you for doing this. Please add tests for KubernetesVersions with pre and build fields.
/ok-to-test

**Comment by johngmyers** [comment]
`Test_ParseKubernetesVersion` has a bunch of test cases to copy.

**Comment by Hellcatlk** [comment]
@johngmyers , thanks for your suggest. I have update the code.

File: pkg/k8sversion/version_test.go [hunk 1]
```diff
@@ -55,3 +57,54 @@ func TestParse(t *testing.T) {
...
+			Expected: true,
+		},
+
+		{
+			Name:     "Version greater than KV",
+			Version:  "1.6.5",
+			Expected: false,
+		},
+		{
+			Name:     "KV equal Version",
```
Comment (justinsb) [inline pkg/k8sversion/version_test.go:96]: Nit: I think Name should be "Version greater than KV" (?)

File: pkg/k8sversion/version_test.go [hunk 2]
```diff
@@ -55,3 +57,54 @@ func TestParse(t *testing.T) {
...
+			Version:  "1.6.5+ea69570f61af8e",
+			Expected: false,
+		},
+	}
+
+	for _, c := range cases {
+		t.Run(c.Name, func(t *testing.T) {
+			ret := kv.IsGTE(c.Version)
```
Comment (justinsb) [inline pkg/k8sversion/version_test.go:104]: Nit: I normally use `actual` for the value we got

**Comment by justinsb** [comment]
Thanks @Hellcatlk - one nit, one tip, but more tips for next time!

/approve
/lgtm

**Pull request merged**